spec/components/markdown_editor_spec.rb

```diff
@@ -9,8 +9,8 @@
 
   it "renders a textarea with a label and toolbar" do
     render "components/markdown_editor",  label: {
-                                            text: "Body",
```

Comment:
was this intended?

Reply:
Maybe the person who added it thought it looked better? But rubocop was complaining so I removed it. 🤷

Reply:
Ah I see sorry - yeah, have tidied that up! Thanks 👍